autoria/api.py

```diff
@@ -334,34 +334,61 @@ def __init__(self, category: str, mark: str, model: str,
         # Getting list of categories and selecting needed id
         category_id = select_item(category, self._api.get_categories())
         mark_id = select_item(mark, self._api.get_marks(category_id))
-        state_id = select_item(state, self._api.get_states())
+        model_id = select_item(
+            model,
+            self._api.get_models(category_id, mark_id)
+        )
 
-        self._params = {
-            'main_category': category_id,
-            'body_id': select_item(
+        state_id, body_id, city_id, gear_id, option_ids, \
+            fuel_ids, drive_ids, color_id = (None,) * 8  # 8 - qty of variables
```

Comment:
Okay, I'll rework the code this way... Tbh the main reason of implementing these variables was me trying to avoid having a lot of code like `self._params ['key'] = value` because that's a lot of typing :sweat_smile: plus code looks less clear.